phpstan.neon.dist

```diff
@@ -11,6 +12,7 @@ parameters:
         - vendor
         - proto
         - tests/TraceContext/W3CTestService
+        - examples # TODO: Remove this once examples are updated
```

Comment:
I excluded examples directory temporarily from phpstan and psalm just to cut down on the scope of an already huge PR. Can do a pass on the examples in a follow up PR.